docs/proposals/202106120-edgex-integration.md

```diff
@@ -80,25 +80,235 @@ As the deployment topology diagram shows.  Each components in the EdgeX instance
+                      Name is NodePool Name
+                    type: string
+                  nodeSelectorTerm:
+                    description: Indicates the node selector to form the pool. Depending
+                      on the node selector, pods provisioned could be distributed
+                      across multiple groups of nodes. A pool's nodeSelectorTerm is
+                      not allowed to be updated.
+                    type: object
+                  patch:
+                    description: Indicates the patch for the templateSpec Now support
+                      strategic merge path :https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/#notes-on-the-strategic-merge-patch
+                      Patch takes precedence over Replicas fields If the Patch also
+                      modifies the Replicas, use the Replicas value in the Patch
+                    type: object
+                  replicas:
+                    description: Indicates the number of the pod to be created under
+                      this pool.
+                    format: int32
+                    type: integer
+                  tolerations:
+                    description: Indicates the tolerations the pods under this pool
+                      have. A pool's tolerations is not allowed to be updated.
+                    items:
+                      description: The pod this Toleration is attached to tolerates
+                        any taint that matches the triple <key,value,effect> using
+                        the matching operator <operator>.
+                      type: object
+                    type: array
+                required:
+                - name
+                type: object
+              version:
+                type: string
+            type: object
+          status:
+            description: EdgeXStatus defines the observed state of EdgeX
+            properties:
+              componetsstatus:
+                items:
+                  properties:
+                    deploymentstatus:
+                      description: DeploymentStatus is the most recently observed
+                        status of the Deployment.
+                      type: object
+                  type: object
+                type: array
+              initialized:
+                type: boolean
+              message:
+                description: A human readable message indicating details about the
+                  transition.
+                type: string
+              reason:
+                description: The reason for the condition's last transition.
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
 
 ```
 
 
 ```go
+type PoolSpec struct {
```

Comment:
in UnitedDeployment definition, field `Replicas` and `Patch` is used for one kind Deployment, but in EdgeX there are several Deployments.  I think it's not fitted to all deployments with same `Replicas` and `Patch`.